code/frodo/Hacl.Frodo.Random.fst

```diff
@@ -14,11 +14,14 @@ module S = Spec.Frodo.Random
 
 friend Lib.IntTypes
 
+#set-options "--z3rlimit 50 --fuel 0 --ifuel 0"
+
 let state = gcmalloc HyperStack.root 0uy 48ul
 
 let randombytes_init_ entropy_input =
   let h0 = HyperStack.ST.get () in
-  assume (as_seq h0 state == S.randombytes_init_ (as_seq h0 entropy_input))  
+  assume (as_seq h0 state == S.randombytes_init_ (as_seq h0 entropy_input))
```

Comment:
would be nice to add a comment here in case someone reviews this code to explain the assumption (i.e. that we can't "realize" randomness and thus have to assume it)

Reply:
note: with @Kachoc's most recent PR, you *may* be able to do something about that (but I'm not sure)

Reply:
I'm not sure either, but I know that for testing we need to drop the implementation of this function in favor of https://github.com/microsoft/PQCrypto-LWEKE/blob/669522db63850fa64d1a24a47e138e80a59349db/tests/rng.h#L70. It seems fine to me to leave it as is.